src/containers/HomePage/tablePanel.tsx

```diff
@@ -0,0 +1,99 @@
+import React, {useEffect, useState} from 'react';
+import {FetchBaseQueryError} from '@reduxjs/toolkit/dist/query';
+
+import {Button, Loader, Table} from '@fluentui/react-northstar';
+
+import {APIError, ConnectedChannelTableData} from '../../types';
+
+import {useGetConnectedChannelsQuery} from '../../services';
+
+const TablePanel = () => {
+    // States
+    const [rowsData, setRowsData] = useState<ConnectedChannelTableData[]>([]);
+
+    // Table headers
+    const header = {
+        items: ['MS Teams Channel', 'Mattermost Channel', 'Action'],
+    };
+
+    // Services
+    const {isLoading, data, isError, error, isFetching} = useGetConnectedChannelsQuery();
```

Comment:
what's the difference between `isLoading` and `isFetching`?

Reply:
`isLoading` changes on the first API request when there is no data present whereas `isFetching` for subsequent requests irrespective of whether data is present or not

Reply:
can't we keep only `isFetching`?